.github/workflows/release.yml

```diff
@@ -28,6 +28,12 @@ jobs:
           profile: minimal
           override: true
 
+      - name: Update Cargo.lock
+        uses: actions-rs/cargo@v1
+        with:
+          command: update
+          args: -w
+
```

Comment:
The version in `Cargo.toml` won't actually be updated until `npx semantic-release` has run. Likewise, the `git commit` which consumes the list of files in `assets` in `package.json` is run _within_ `semantic-release`, so moving this command to after that one wouldn't have the desired effect either.

I _think_ what needs to happen is putting this command inline in the `@semantic-release/exec` plugin.